includes/definitions/discovery/qnap.yaml

```diff
@@ -69,10 +69,10 @@ modules:
                     state_name: hdStatus
                     states:
                         - { value: 0, generic: 0, graph: 0, descr: 'Ready' }
-                        - { value: -5, generic: 3, graph: -5, descr: 'No Disk' }
-                        - { value: -6, generic: 2, graph: -6, descr: 'Invalid' }
-                        - { value: -9, generic: 2, graph: -9, descr: 'RW Error' }
-                        - { value: -4, generic: 3, graph: -4, descr: 'Unknown' }
+                        - { value: -5, generic: 3, graph: 0, descr: 'No Disk' }
```

Comment:
It would be cool if someone could document what `graph` values actually mean :)

Reply:
Yeah, that way I would not did it wrong in the first place!!! hahaha

I dont have idea on what graph means. I just made the synology move to yaml discovery and found that I did it wrong in the qnap.

Reply:
graph does nothing :D  AFAIK it was "for future use"